spec/rails_helper.rb

```diff
@@ -66,10 +66,8 @@
   config.include FactoryBot::Syntax::Methods
   config.include HmisCsvFixtures
 
-  config.include FixpointTestHelpers
-
   config.before(:suite) do
-    Dir.glob('{drivers,spec}/**/fixpoints/*.yml').each do |filename|
+    Dir.glob('{drivers,spec}/**/fixpoints/*.sql').each do |filename|
```

Comment:
Should this be consistent w/ `.gitignore`, and either process both `.yml` and `.sql`, or remove `.yml` from above?